test/test_helper.rb

```diff
@@ -31,6 +31,11 @@
   c.strict_keyword_argument_matching = true
 end
 
+Rubygem.__elasticsearch__.create_index! force: true
+Minitest.after_run do
+  Rubygem.__elasticsearch__.delete_index!
```

Comment:
Do we need to delete the ElasticSearch index after the test suite has finished? Having the search index around would be convenient if there’s a bug/failing search-related test.

Reply:
Good question. I was operating under the assumption that I needed to delete the index since that's what I had to do to find the order dependent test failure. Now that it's found, I think you're right that I can remove the line that deletes the index now.

As for debugging, the index gets "refreshed" regularly during the run. I'm not precisely sure how that affects the index but you might need to stop it mid-run to see anything useful if just one test fails.

I'll update this PR today.

Reply:
Updated.